docs/_docs/versions.md

```diff
@@ -5,114 +5,157 @@ layout: post-toc
 redirect_from: /docs/
 ---
 
-# Manage Versions of Your Zapier Integration
+# How to update your integration on the Zapier Developer Platform
 
-Your Zapier integration should be quite nice when first launched—minimal, perhaps, but with the basics covered. The app authentication will let users connect to your API, and the triggers and actions should get data to and from your API.
+## Overview
 
-Over time, though, things will change. Users will request new triggers and actions, or want to send and receive more data with your existing ones. You’ll add new features to your app and want to add them to your Zapier integration, too. And eventually, your API may change, and your Zapier integration’s authentication or API calls may need to be changed as well.
+Later sections of this guide will cover the steps for _Cloning_, _Promoting_, _Migrating_, and _Deprecating_ integration versions in more detail. Put briefly, the process of creating a new version of your integration takes three steps in either the Developer Platform [Visual Builder](https://developer.zapier.com/) or [Zapier Command Line Interface](https://platform.zapier.com/docs/export) (CLI).
```

Comment:
>https://platform.zapier.com/docs/export

Since the audience here is VB users, is the intent here that VB users would consider migrating to CLI? 

Or would it be more appropriate to link the CLI versions tutorial: https://platform.zapier.com/cli_tutorials/versions

Reply:
Concur with this - the docs in this area are primarily intended for VB users. I would recommend explicitly suggesting that CLI users consult a different doc, although we could mention that the steps are similar even if the mechanism is different.

Since it seems like you wrote a lot of the later part of the doc with a changed audience in mind, maybe this is something we need to discuss in more detail before we dig in on reviewing the exact changes?

I'd also suggest we retain some of the previous content about what versions are for. Starting an overview with "what I'm going to address later" is a bit odd, and a lot of people who use the VB are not normally developers and the concepts of versioning may not be familiar to them, so this content is potentially useful.